jkube-kit/remote-dev/src/test/java/org/eclipse/jkube/kit/remotedev/RemoteDevelopmentServiceTest.java

```diff
@@ -67,18 +78,44 @@ void canBeStoppedMultipleTimesBeforeStart() {
   @Test
   @DisplayName("start initiates PortForwarder and KubernetesSshServiceForwarder")
   void startInitiatesChildProcesses() {
+    // When
+    startDevelopmentServiceWithConfig(RemoteDevelopmentConfig.builder().build());
+
+    // Then
+    verify(logger, times(1))
+      .debug("Creating or replacing Kubernetes services for exposed ports from local environment");
+    verify(logger, times(1))
+      .debug("Starting Kubernetes SSH service forwarder...");
+    verify(logger, times(1))
```

Comment:
If it's verifying for just one time invocation, maybe you can omit `times(1)`
```suggestion
    verify(logger)
```

Reply:
please, keep times(1), we are using the verbose style all across the code-base to make it clearer (especially when reviewing on GH).